ansible_base/authenticator_plugins/base.py

```diff
@@ -16,6 +17,7 @@ class BaseAuthenticatorConfiguration(serializers.Serializer):
         help_text="Any additional fields that this authenticator can take, they are not validated and passed directly back to the authenticator",
         required=False,
         allow_null=True,
+        ui_field_label='Additional Authenticator Fields',
```

Comment:
Can/should we start _()ing new user-facing strings? We already import `_` above.